projects/storefrontlib/shared/components/captcha/captcha.component.ts

```diff
@@ -64,12 +75,34 @@ export class CaptchaComponent implements AfterViewInit, OnDestroy {
             })
           )
           .subscribe(() => {
-            this.confirmed.emit(true);
+            setTimeout(() => {
```

Comment:
can't get the points of using timeout and ChangeDetectorRef

Reply:
Remove timeout and ChangeDetectorRef.